client/src/locales/en/translation.json

```diff
@@ -2,5 +2,8 @@
   "Hello": "Hello",
   "Do you know this word?": "Do you know this word?",
   "YES": "YES",
-  "NO": "NO"
+  "NO": "NO",
+  "Show next": "Show next word >",
+  "Give the cue": "Give the cue"
+
```

Comment:
Could you double-check if this file formatted?

Reply:
done